UPGRADE.md

```diff
@@ -104,6 +104,10 @@ After adding native & logical types into the Flow, we remove the `CollectionEntr
 
 Removed `BooleanEntry::from()`, `FloatEntry::from()`, `IntegerEntry::from()`, `StringEntry::fromDateTime()` methods in favor of using DSL functions.
 
+### 14) Removed deprecated `Sha1IdFactory`
+
+Class `Sha1IdFactory` was removed, use `HashIdFactory` class now.
```

Comment:
I think that before we can fully remove it, we should allow users to change the hashing algorithm in the HashIdFactory in case someone was already using Sha1.

Reply:
Reverted, will extract into separated PR.